CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Extend case search to include internal MongoDB id
 - Support for connecting to a MongoDB replica set (.py config files)
 - Support for connecting to a MongoDB replica set (.yaml config files)
+- Added function for maintainers and admins to remove gene panels
+- Admins can restore deleted gene panels
```

Comment:
Ok! I wasn't really familiar with the functionality. So admins and maintainers can remove maintained panels but normal users can't? Perhaps this could be explained a bit better in the changelog?